README.md

```diff
@@ -0,0 +1,52 @@
+# ProtoDash
+
+ProtoDash is a tool to aid the rapid development of prototype dashboards and enable data engineering and data science to deploy static sites without the need to engage with ops.
+
+## Usage
```

Comment:
```suggestion
## Local Development
```

I think mostly people won't need to do this, the exception being if someone in data engineering/operations wants to add a feature. Can you move this down, maybe just before "Environment Config"?